menu.json

```diff
@@ -708,62 +708,79 @@
-                    "text": "Our People"
-                },
-                {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/Employment/",
-                    "text": "Employment Opportunities",
-                    "cssClass": "featVideo"
+                    "text": "Who We Are",                
+                    "breakListBefore": true,
+                    "children": [
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/people/",
+                            "text": "Our People",
+                            "cssClass": "Popular"
+                        },
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/ssw/Employment/",
+                            "text": "Employment Opportunities",
+                            "cssClass": "Popular"
+                        }
+                    ]
                 },
                 {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/Company/Clients",
-                    "text": "Our Clients",
-                    "cssClass": "featVideo",
+                    "text": "Who We Work With",
                     "children": [
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/ssw/Company/Clients",
+                            "text": "Our Clients"
+                        },
                         {
                             "navigateUrl": "https://www.ssw.com.au/ssw/Testimonials/default.aspx",
                             "text": "Testimonials"
                         }
                     ]
                 },
                 {
-                    "navigateUrl": "https://www.ssw.com.au/ssw/Company/Offices/",
-                    "text": "Our Offices",
+                    "text": "What We Say",                
+                    "breakListBefore": true,
+                    "children": [
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/ssw/News/",
+                            "text": "News & Press"
+                        },
+                        {
+                            "navigateUrl": "https://blog.ssw.com/",
+                            "text": "Blog"
+                        },
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/ssw/NETUG/SSWUpdatePrevious.aspx",
+                            "text": "Newsletters"
+                        }
+                    ]
+                },
+                {
+                    "text": "Where To Find Us",                
                     "breakListBefore": true,
+                    "cssClass": "map",
                     "children": [
+                        {
+                            "navigateUrl": "https://www.ssw.com.au/ssw/Company/Offices/",
+                            "text": "All Offices"
+                        },
```

Comment:
As per the email, we decided to remove this link from the menu

> Where To Find Us (Pin)
> - SSW Sydney
> - SSW Melbourne
> - SSW Brisbane
> - SSW Newcastle
> - SSW Hangzhou
> - Contact Us (Popular)

```suggestion

```

Reply:
As per my conversation with Adam last night we decided to add that, otherwise we lose the link to the offices page.

Reply:
As per our conversation on teams, we have agreed to keep this link